components/homme/cmake/machineFiles/weaver.cmake

```diff
@@ -22,25 +49,32 @@ set(HOMMEXX_VECTOR_SIZE 1 CACHE STRING "")
 set(HOMME_FIND_BLASLAPACK TRUE CACHE BOOL "")
 set(USE_QUEUING FALSE CACHE BOOL "")
 set(USE_TRILINOS OFF CACHE BOOL "")
-set(WITH_PNETCDF FALSE CACHE FILEPATH "")
-set(USE_NUM_PROCS 1 CACHE STRING "Num mpiprocs to use")
+#set(WITH_PNETCDF FALSE CACHE FILEPATH "")
+set(USE_NUM_PROCS 4 CACHE STRING "Num mpiprocs to use")
 set(HAVE_EXTRAE TRUE CACHE BOOL "")
-set(Extrae_LIBRARY "-L${NETCDF_DIR}/lib -L${NetCDF_Fortran_PATH}/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5 -ldl -lz" CACHE STRING "")
 set(ENABLE_COLUMN_OPENMP OFF CACHE BOOL "")
 set(ENABLE_HORIZ_OPENMP OFF CACHE BOOL "")
 set(HOMMEXX_EXEC_SPACE "CUDA" CACHE STRING "")
 
+
+SET(BUILD_HOMME_PREQX_KOKKOS TRUE CACHE BOOL "")
+SET(BUILD_HOMME_THETA_KOKKOS TRUE CACHE BOOL "")
+#sET(HOMME_ENABLE_COMPOSE FALSE CACHE BOOL "")
+
+set(HOMMEXX_BFB_TESTING TRUE CACHE STRING ON)
+
 # Kokkos settings
 set(ENABLE_OPENMP FALSE CACHE BOOL "")
-set(KOKKOS_ENABLE_DEBUG FALSE CACHE BOOL "")
-set(KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
-set(KOKKOS_ENABLE_CUDA TRUE CACHE BOOL "")
-set(KOKKOS_ENABLE_CUDA_LAMBDA TRUE CACHE BOOL "")
-set(KOKKOS_ARCH Volta70 CACHE STRING "")
-set(KOKKOS_ENABLE_DEPRECATED_CODE FALSE CACHE BOOL "")
-set(KOKKOS_ENABLE_EXPLICIT_INSTANTIATION FALSE CACHE BOOL "")
+set(Kokkos_ENABLE_DEBUG FALSE CACHE BOOL "")
+set(Kokkos_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
+set(Kokkos_ENABLE_CUDA TRUE CACHE BOOL "")
+set(Kokkos_ENABLE_CUDA_LAMBDA TRUE CACHE BOOL "")
+set(Kokkos_ARCH_VOLTA70 TRUE CACHE BOOL "")
+set(Kokkos_ENABLE_DEPRECATED_CODE FALSE CACHE BOOL "")
+set(Kokkos_ENABLE_EXPLICIT_INSTANTIATION FALSE CACHE BOOL "")
 
 # Compilers
 set(CMAKE_C_COMPILER mpicc CACHE STRING "")
 set(CMAKE_Fortran_COMPILER mpif90 CACHE STRING "")
-set(CMAKE_CXX_COMPILER mpicxx CACHE STRING "")
+######## CHANGE THAT
```

Comment:
Are you going to revert this back to mpicxx before merging?

Reply:
no, why? this is more to draw attn. so that someone else points to their mvcc_wrapper.

Reply:
Because the compiler should always be mpicxx. It's the env var `OMPI_CXX` (or MPICH_CXX for mpich distros) that should point to nvcc_wrapper. Setting the compiler to be nvcc_wrapper can be problematic for MPI.